src/constants/WalletAndCurrencyConstants.ts

```diff
@@ -509,6 +513,7 @@ export const SPECIAL_CURRENCY_INFO: {
     chainCode: 'OP',
```

Comment:
`chainCode: 'OP',` must have been a copy paste but. Can you fix it while we're here.